pkgs/sdk/server/src/Internal/FDv2Payloads/FDv2Event.cs

```diff
@@ -52,6 +52,23 @@ public FDv2Event(string eventType, JsonElement jsonData)
             JsonData = jsonData;
         }
 
+        public static bool TryDeserializeFromJsonString(string eventType, string jsonString, out FDv2Event evt, out string error)
```

Comment:
Allow for easy deserialization of the payload from streaming, where we start with a string.